upt_macports/templates/ruby.Portfile

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.Portfile' %}
+
+{% block portgroup %}
+PortGroup           ruby 1.0
+{% endblock %}
+{% block nameversion %}
+ruby.setup          {{ pkg._pkgname() }} {{ pkg.upt_pkg.version }} gem {} rubygems
+revision            0
+{% endblock %}
+
+{% block versions %}
+{%- if pkg.upt_pkg.requirements.run|length > 0 %}
```

Comment:
using the ```ruby.branches``` approach it will generate subports similarly to the situation for Perl/Python, so for example rb25-<name> and rb26-<name>. I'd imagine that we use the same ```if {${subport} ne ${name}} {``` block as in the Python ports here as well.